### PR TITLE
Refactor the documentation about the SUMO route file creation.

### PR DESCRIPTION
**Pull request by fabienrohrer**
Ditto

### CODE REVIEW

File: automobile/scenario-creation-tutorial.md [hunk 1]
```diff
@@ -68,7 +68,38 @@ To do this, open the `sumo.net.xml` file in SUMO `netedit`:
...
 ```
 
-You can then generate the route file, SUMO provides several ways to generate route files. You may for example generate a [flow file](http://sumo.dlr.de/wiki/Definition_of_Vehicles,_Vehicle_Types,_and_Routes) and then use [duarouter](http://sumo.dlr.de/wiki/DUAROUTER) to generate the route file for you:
+
+### Generate the SUMO route file
+
+At this stage, a SUMO route file should be created to describe the traffic, i.e.,
+the number of vehicles, which road they should take, the vehicle properties, etc.
+
+This [SUMO document](http://sumo.dlr.de/wiki/Definition_of_Vehicles,_Vehicle_Types,_and_Routes) explains precisely how to achieve this.
```
Comment (tn12787) [inline automobile/scenario-creation-tutorial.md]: I'd maybe say that the sumo document "explains in detail how this is done"

File: automobile/scenario-creation-tutorial.md [hunk 2]
```diff
@@ -68,7 +68,38 @@ To do this, open the `sumo.net.xml` file in SUMO `netedit`:
...
+
+This following subsections describes two ways to create a SUMO route file.
+The use of the SUMO [duarouter](http://sumo.dlr.de/wiki/DUAROUTER) program is required in both cases.
+
+
+#### Create randomly a SUMO route file
```
Comment (tn12787) [inline automobile/scenario-creation-tutorial.md]: I'd move "randomly" to then end of this heading e.g. "Create a SUMO route file randomly , same for the following heading.

File: automobile/scenario-creation-tutorial.md [hunk 3]
```diff
@@ -68,7 +68,38 @@ To do this, open the `sumo.net.xml` file in SUMO `netedit`:
...
-You can then generate the route file, SUMO provides several ways to generate route files. You may for example generate a [flow file](http://sumo.dlr.de/wiki/Definition_of_Vehicles,_Vehicle_Types,_and_Routes) and then use [duarouter](http://sumo.dlr.de/wiki/DUAROUTER) to generate the route file for you:
+
+### Generate the SUMO route file
+
+At this stage, a SUMO route file should be created to describe the traffic, i.e.,
+the number of vehicles, which road they should take, the vehicle properties, etc.
+
+This [SUMO document](http://sumo.dlr.de/wiki/Definition_of_Vehicles,_Vehicle_Types,_and_Routes) explains precisely how to achieve this.
+
+This following subsections describes two ways to create a SUMO route file.
```
Comment (tn12787) [inline automobile/scenario-creation-tutorial.md]: "This following subsections describes" should read "The following subsections describe".

File: automobile/scenario-creation-tutorial.md [hunk 4]
```diff
@@ -68,7 +68,38 @@ To do this, open the `sumo.net.xml` file in SUMO `netedit`:
...
+
+```sh
+python $WEBOTS_HOME/projects/automobile/resources/sumo/tools/randomTrips.py -n $WBT_PROJECT_PATH/worlds/myMap_net/sumo.net.xml -o $WBT_PROJECT_PATH/worlds/myMap_net/sumo.trip.xml -e 3600 -p 2
+$WEBOTS_HOME/projects/automobile/resources/sumo/bin/duarouter --trip-files $WBT_PROJECT_PATH/worlds/myMap_net/sumo.trip.xml --net-file $WBT_PROJECT_PATH/worlds/myMap_net/sumo.net.xml --output-file $WBT_PROJECT_PATH/worlds/myMap_net/sumo.rou.xml --ignore-errors true
+```
+
+
+#### Create manually a SUMO route file
+
+Alternatively a SUMO flow file can be created to define the traffic by vehicles flows.
```
Comment (tn12787) [inline automobile/scenario-creation-tutorial.md]: "vehicles" shouldn't be plural here.

**Comment by fabienrohrer** [comment]
Thank you, I think I fixed every thing.

**Comment by fabienrohrer** [comment]
👍

File: automobile/scenario-creation-tutorial.md [hunk 5]
```diff
@@ -68,7 +68,38 @@ To do this, open the `sumo.net.xml` file in SUMO `netedit`:
...
+
+
+#### Create randomly a SUMO route file
+
+The purpose of the SUMO `randomTrips.py` script is to generate a random SUMO trip file.
+This file can be used as an input argument of `SUMO duarouter` to create the expected SUMO route file.
+Here are the commands to do so:
+
+```sh
+python $WEBOTS_HOME/projects/automobile/resources/sumo/tools/randomTrips.py -n $WBT_PROJECT_PATH/worlds/myMap_net/sumo.net.xml -o $WBT_PROJECT_PATH/worlds/myMap_net/sumo.trip.xml -e 3600 -p 2
```
Comment (DavidMansolino) [inline automobile/scenario-creation-tutorial.md]: Really sory for the late review...

Comment (fabienrohrer) [inline automobile/scenario-creation-tutorial.md]: No problems. But honestly, I think the best is to simply drop these arguments.

Indeed, according to http://sumo.dlr.de/wiki/Tools/Trip , `-e 3600` is already the default value. And `-p 1` is the default value, but it won't change the result a lot. All the SUMO scripts are full of parameters, if a user want to change them, he needs to read the SUMO documentation.

Comment (DavidMansolino) [inline automobile/scenario-creation-tutorial.md]: I fully agree, it is indeed better to simply remove them.

**Pull request merged**